x-pack/plugins/enterprise_search/server/routes/enterprise_search/indices.ts

```diff
@@ -577,12 +577,17 @@ export function registerIndexRoutes({
         pipeline: { description: defaultDescription, ...pipeline },
       };
 
-      const simulateResult = await client.asCurrentUser.ingest.simulate(simulateRequest);
+      try {
+        const simulateResult = await client.asCurrentUser.ingest.simulate(simulateRequest);
 
-      return response.ok({
-        body: simulateResult,
-        headers: { 'content-type': 'application/json' },
-      });
+        return response.ok({
+          body: simulateResult,
+          headers: { 'content-type': 'application/json' },
+        });
+      } catch (e) {
+        log.error(`Error simulating inference pipeline: ${JSON.stringify(e)}`);
```

Comment:
Is this ok? I added it as I kept having to add it while testing locally to diagnose failures.